src/index.d.ts

```diff
@@ -4,6 +4,15 @@ interface SnippetOptions {
   debug?: boolean;
   host?: string;
   script?: string;
+  iFrameOption?: IFrameOption;
+}
+
+// recording options for iframes
+// `RecordCrossDomainIFrame` has security implications
+// see https://help.fullstory.com/hc/en-us/articles/360020622514
+export enum IFrameOption {
```

Comment:
Would it be a good idea to call this `IFrameOptions` plural?

Reply:
Yes.